app/views/contexts/list_context.py

```diff
@@ -95,12 +96,17 @@ def _build_list_items_context(
 
             yield list_item_context
 
-    def _get_item_title(self, summary_definition, list_item_id, is_primary):
+    def _get_item_title(
+        self,
+        summary_definition: Mapping[str, Any],
+        list_item_id: Optional[str],
+        is_primary: bool,
+    ) -> Any:
         rendered_summary = self._placeholder_renderer.render(
             summary_definition, list_item_id
         )
 
         if is_primary:
-            rendered_summary["item_title"] += lazy_gettext(" (You)")
+            rendered_summary["item_title"] += lazy_gettext(" (You)")  # type: ignore
```

Comment:
add a comment to explain why we added the `#type :ignore`

Reply:
Added 👍